include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -671,19 +671,30 @@ def LowerXMR : Pass<"firrtl-lower-xmr", "firrtl::CircuitOp"> {
   ];
 }
 
-def LowerIntrinsics : Pass<"firrtl-lower-intrinsics", "firrtl::CircuitOp"> {
-  let summary = "Lower intrinsics";
+def IntrinsicInstancesToOps : Pass<"firrtl-intrinsic-instances-to-ops", "firrtl::CircuitOp"> {
+  let summary = "Lower instances instances of intrinsic modules to ops.";
   let description = [{
-    This pass lowers intrinsics encoded as extmodule with annotation and
-    intmodule to their implementation or op.
+    This pass replaces instances of intrinsic modules (intmodule) with 
+    `firrtl.int.generic` operations, and removes the intmodule's from
+    the circuit.
+ 
+    Use this before running LowerIntrinsics.
   }];
-  let constructor = "circt::firrtl::createLowerIntrinsicsPass()";
+  let constructor = "circt::firrtl::createIntrinsicInstancesToOpsPass()";
   let options = [
     Option<"fixupEICGWrapper", "fixup-eicg-wrapper", "bool", "false",
       "Lower `EICG_wrapper` modules into clock gate intrinsics">,
   ];
 }
 
+def LowerIntrinsics : Pass<"firrtl-lower-intrinsics", "firrtl::FModuleOp"> {
```

Comment:
It's very nice that this gets to be a FIRRTL module pass now, instead of a circuit pass. :partying_face: